aspire-orchestrator/README.md

```diff
@@ -1,54 +1,88 @@
-# .NET Aspire Integratiuon
+# .NET Aspire Integration
 
 ## Run Prerequisites
 
-- [dotnet (8 or greater)](https://dotnet.microsoft.com/en-us/download)
+- [dotnet (8 or greater)](https://dotnet.microsoft.com/download)
+- [Python 3.8 or greater](https://www.python.org/downloads)
+- [NodeJs v18.12 or greater](https://nodejs.org/en/download)
 
 ## Deployment Prerequisites
 
-- [Azure Developer CLI](https://learn.microsoft.com/en-us/azure/developer/azure-developer-cli/install-azd?tabs=winget-windows%2Cbrew-mac%2Cscript-linux)
-- [dotnet (9 or greater)](https://dotnet.microsoft.com/en-us/download)
-- [Python 3.8 or greater](https://www.python.org/downloads/)
-- [NodeJs v18.12 or greater](https://nodejs.org/en/download/)
+- [Azure Developer CLI](https://learn.microsoft.com/azure/developer/azure-developer-cli/install-azd)
+- [Docker for Desktop](https://docs.docker.com/get-started/introduction/get-docker-desktop)
+- [dotnet (9 or greater)](https://dotnet.microsoft.com/download)
+- [Python 3.8 or greater](https://www.python.org/downloads)
+- [NodeJs v18.12 or greater](https://nodejs.org/en/download)
 
 ## Run with .NET Aspire
 
-1. Make sure you have accepted the [dotnet trust dev certificate](https://learn.microsoft.com/en-us/aspnet/core/security/enforcing-ssl?view=aspnetcore-9.0&tabs=visual-studio%2Clinux-sles#trust-the-aspnet-core-https-development-certificate)
-```bash
-dotnet dev-certs https --trust
-```
+1.  Make sure your workstation trusts
+    [ASP.NET Core HTTPS development certificate](https://learn.microsoft.com/en-us/aspnet/core/security/enforcing-ssl?#trust-the-aspnet-core-https-development-certificate)
 
-2. Clone the repository
+        dotnet dev-certs https --trust
 
-```bash
-make
-cd aspire-orchestrator
-cd Aspire.AppHost
-dotnet run
-```
+2.  Clone the repository
+
+        make
+        cd aspire-orchestrator
+        cd Aspire.AppHost
+        dotnet run
 
 ## Deployment Steps with azd
 
-1. Clone the repository
-2. In the root folder, run
-```bash
-make
-```
-3. [Configure the app registration and configure workbenchapp and workbench service](../docs/CUSTOM_APP_REGISTRATION.md). Set the values in the `aspire-orchestrator/aspire-orchestrator/aspire-orchestrator/aspire-orchestrator/appsettings.json` file
-```json
-{
-    "EntraID": {
-        "ClientId": "<CLIENT_ID>",
-        "Authority": "https://login.microsoftonline.com/<TENANT_ID>"
-    }
-}
-```
-4. Authenticate with Azure Developer CLI
-```bash
-azd login
-```
-5. Create the infrastructure and deploy the application
-```bash
-azd up
-```
-6. Update the App Registration with the correct workbenchapp redirect URI
\ No newline at end of file
+1.  Clone the repository
+
+2.  [Configure the Entra app registration](../docs/CUSTOM_APP_REGISTRATION.md).
+    Copy the Entra **Application Id** and **Tenant Id** into
+    `aspire-orchestrator/Aspire.AppHost/appsettings.json` file.
```

Comment:
Is there a way these values could be pulled in from something like a .env that is gitignored?

Reply:
I believe the proper approach is overriding with a `appsettings.Development.json` and `appsettings.Production.json` - something to be tested

Reply:
Ah, so those are ignored as they are deployment configurations? Do you pass them in with a flag or something?

Reply:
appsettings are retrieved automatically by all .NET apps using the Web and Aspire SDK. When you run `azd up`, Aspire goes and fetch values from these files.